lib/checks/lists/listitem.js

```diff
@@ -1,18 +1,21 @@
 const parent = axe.commons.dom.getComposedParent(node);
 if (!parent) {
-	return false;
+	// Can only happen with detached DOM nodes and roots:
```

Comment:
should we do similar check for `dlitem` ?

Reply:
Could have. If I wasn't already in the process of changing these functions I would.